python-backend/tests/mines/resources/test_mine_type_detail_resource.py

```diff
@@ -50,7 +80,23 @@ def test_post_mine_type_detail_missing_mine_disturbance_code(test_client, auth_h
     assert post_data == {
         'error': {
             'status': 400,
-            'message': 'Error: Missing mine_disturbance_code.'
+            'message': 'Error: Missing mine_disturbance_code or mine_commodity_code.'
```

Comment:
We should not be asserting the exact error message unless it's defined as a constant or in some central object. Which we haven't does as a pattern for mds yet.  Should only be asserting status == 400.

Reply:
I don't necessarily disagree, but this is the pattern other tests were following when I joined the project. Removing the error message check could also considerably reduce our test coverage

Reply:
Would it reduce our test coverage? the src that adds the message is still being executed even if the test isn't asserting it's value. We can have a larger discussion, but should changing the error message really cause a test to fail. and if the exact strings error messages should be is a high priority item, then we need to centrally define them so that we can test them (assert message = ERROR_MESSAGES['mine_type'][101]) or something like that.

Reply:
This might need a larger discussion, but I believe we should not be checking that error messages are exacting a certain string. Maybe just check if it isn't null? 

I think it adds alot of fragility and maintenance effort when the test should be 'did it fail' not 'did it fail with this exact error message'.

Reply:
As I said, I agree. And I will gladly switch them all over to use constants when we've made that switch. I'll even write the constants. I just don't see that as in-scope for this PR. But this test coverage is in-scope since the specific error messages are part of the expected behaviour

Reply:
I would tend to disagree, but that refactor would be outside of the scope of the PR. I'll approve it as you are following the pattern and I'll make a note to bring this up at the end of sprint.